[Swift] fastlane error for Deliverfile

All I did was run `fastlane init swift` and trying to run this. Fails because of `Deliverfile`. I do have a `Deliverfile.swift` 👍 

```
▸ Compiling PrecheckfileProtocol.swift
▸ Compiling SocketResponse.swift
▸ Compiling main.swift
▸ Compiling Scanfile.swift
▸ Compiling Snapshotfile.swift
▸ Compiling Fastlane.swift

❌  /Users/felixkrause/Developer/hacking/majorkey/fastlane/swift/Fastlane.swift:4065:18: use of undeclared type 'Deliverfile'

let deliverfile: Deliverfile = Deliverfile()
                 ^~~~~~~~~~~


▸ Compiling Screengrabfile.swift
** BUILD FAILED **


The following build commands failed:
CompileSwift normal x86_64 /Users/felixkrause/Developer/hacking/majorkey/fastlane/swift/Fastlane.swift
CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
[18:05:56]: Exit status: 65
+-----------+---------+
|    Lane Context     |
+-----------+---------+
| LANE_NAME | release |
+-----------+---------+
[18:05:56]: Exit status: 65
[18:05:56]: fastlane finished with errors

[!] Exit status: 65
```

--- a/deliver/lib/assets/DeliverfileDefault.swift
+++ b/deliver/lib/assets/DeliverfileDefault.swift
@@ -1,3 +1,13 @@
 // The Deliverfile allows you to store various App Store Connect metadata
 // For more information, check out the docs
 // https://docs.fastlane.tools/actions/deliver/
+
+// In general, you can use the options available
+// fastlane deliver --help
+
+// Remove the // in front of the line to enable the option
+
+class Deliverfile: DeliverfileProtocol {
+  //var username: String { return "" }
+  //var appIdentifier: String? { return "" }
+}

